### ISSUE TITLE
Different "paths" through the survey based on answers at the start?

### ISSUE THREAD

**Issue by King-Mob** [Wobbly-App/wobbly-member-survey#11]
It might be cool if someone who wants to join as a coder has a slightly different order of questions than someone joining as a designer, or workplace organiser. So the survey might have a couple of stages to it, first asking how someone sees themselves (and these categories don't need to be exclusive) and then tailoring the later questions to that.

**Comment by david-ml** [comment]
[SurveyJS](https://surveyjs.io/Documentation/Library/#platforms) seems like a good candidate, since it should support the level of branching we need for that. 

I made a POC in Plunker, but I'll see what you all think before starting a PR.

**Comment by marcaaron** [comment]
This is really nice @david-ml. I'm not sure we need hard paths, but can see flat branches of questions working out nice. something like drop downs with more details/questions expanding based on interests. Does that sound cool @King-Mob ? The more you're interested in contributing the more information we ask for.

**Comment by King-Mob** [comment]
Yeah that sounds like a good balance, I really like what David's put together here.
